### ISSUE TITLE
only special version of http-vue-loader  can  work

### ISSUE THREAD

**Issue by mydaoyuan** [chitoku-k/homomaid.vue#3]
i change `mian.js` ,  then dont work.

before: 
```
'http-vue-loader':
      'https://cdn.rawgit.com/chitoku-k/http-vue-loader/bc8e142332b8f396715ec3fc61fbad03ae2a8040/src/httpVueLoader'
```
it ok.
------------
after:    dont work version
```
'http-vue-loader': '../../node_modules/http-vue-loader/src/httpVueLoader'
```
and i make package.json.

```
"dependencies": {
    "http-vue-loader": "^1.3.5"
  }
```
throw  this error
![image](https://user-images.githubusercontent.com/16152141/48698559-5726bd80-ec22-11e8-8afe-edc8f67ca14a.png)

**Comment by chitoku-k** [comment]
Of course not because this repository depends on the PR of http-vue-loader which is not merged to master.

Do 👍 if you desire this to be released here: https://github.com/FranckFreiburger/http-vue-loader/pull/36

**Comment by mydaoyuan** [comment]
> Of course not because this repository depends on the PR of http-vue-loader which is not merged to master.
> 
> Do 👍 if you desire this to be released here: [FranckFreiburger/http-vue-loader#36](https://github.com/FranckFreiburger/http-vue-loader/pull/36)

cool👍 , but i think, this thing can add readme  tell everyone😃

**Comment by chitoku-k** [comment]
Now that http-vue-loader merged FranckFreiburger/http-vue-loader#59, it enabled implementation of loading a Vue module with just a piece of code  🎉
